pages/transaction/zksync/era/faucet.vue

```diff
@@ -92,6 +122,9 @@ const { eraNetwork } = storeToRefs(useEraProviderStore());
 
 const route = useRoute();
 
+// displays the out of funds message replacing the form to request tokens
+const noFunds = true;
```

Comment:
what do you think about replacing this with an env variable @JackHamer09 ?

Reply:
In perfect case we need to place this into network config. But since its temporary lets leave it here for now